vcpkg/users/platforms/android.md

```diff
@@ -261,7 +261,7 @@ export ANDROID_NDK_HOME=/home/your-account/Android/Sdk/ndk-bundle
 
 ### Example exporting \[jsoncpp]
 
-First, "vcpkg install" the 4 android architectures (it is mandatory to export all 4 of them):
+First, "vcpkg install" the 4 Android architectures (it is mandatory to export all 4 of them):
```

Comment:
```suggestion
First, run "vcpkg install" for all supported Android architectures (it is mandatory to export all of them):
```
We probably want to get rid of the specific number in case it changes.